File: Corte1/Calculadora_reto5/Reto5_Calculadora.cpp
#include <iostream>
#include <cmath>
#include <string>
using namespace std;

int main(){

double numero1 = 0.0;
double numero2 = 0.0;
double resultado = 0.0;
string operacion = "";

cout << "Escriba que operación quiere hacer, por favor escribala como aparece en pantalla: suma, resta, multiplicacion, division, potencia, raiz: \n";
cin >> operacion;
cout << "Escriba el primer numero: ";
cin >> numero1;
cout << "escriba el segundo numero: ";
cin >> numero2;

if (operacion == "suma") {
    resultado = numero1 + numero2;
}
else if (operacion == "resta") {
    resultado = numero1 - numero2;
}
else if (operacion == "multiplicacion") {
    resultado = numero1 * numero2;
}
else if (operacion == "division") {
    resultado = numero1 / numero2;
}
else if (operacion == "potencia") {
    resultado = pow(numero1,numero2);
}
else if (operacion == "raiz") {
    resultado = pow(numero1, 1.0/numero2);
}
else {
    cout << "Input invalido.";
    return 0;
}

cout << "El resultado de la operación es: " << resultado;
}

File: Corte1/Calculadora_reto5/Reto5_Calculadora.py
numero1 = input ("Escriba el primer numero:")
numero2 = input ("Escriba el segundo numero:")
operacion =input ("Escriba la operacion que desea realizar: suma, resta, multiplicacion, division, potencia, raiz: ")

if operacion == "suma":
    resultado = float(numero1) + float(numero2)
    print("El resultado de la suma es: ", resultado)
elif operacion == "resta":
    resultado = float(numero1) - float(numero2)
    print("El resultado de la resta es: ", resultado)
elif operacion == "multiplicacion":
    resultado = float(numero1) * float(numero2)
    print("El resultado de la multiplicacion es: ", resultado)
elif operacion == "division":
    resultado = float(numero1) / float(numero2)
    print("El resultado de la division es: ", resultado)
elif operacion == "potencia":
    resultado = float(numero1) ** float(numero2)
    print("El resultado de la potencia es: ", resultado)
elif operacion == "raiz":
    resultado = float(numero1) ** (1/float(numero2))
    print("El resultado de la raiz es: ", resultado)
else:
    print("Input invalido, por favor ingrese una operacion valida: suma, resta, multiplicacion, division, potencia, raiz")